RELEASE.md

```
@@ -0,0 +1,32 @@
+```
+
+5. Monitor the GitHub Actions workflow to ensure the build and publish succeed.
+6. Add a GitHub Release with a small description of changes from the last release.
+
+**Note:** Tags must start with "v" for the publishing GitHub Action to begin.
+
+## Valid version numbers and their meaning
+
+- For version number, we follow [SemVer](https://semver.org/) (major.minor.patch).
+- For pre-release tags, we follow the [PEP440](https://peps.python.org/pep-0440/) syntax:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)